data/lm/generate_lm.py

```diff
@@ -39,10 +39,13 @@ def main():
                '--prune', '0', '0', '1'
     ])
 
-    # Filter LM using vocabulary of top 500k words
-    filtered_path = os.path.join(tmp, 'lm_filtered.arpa')
     vocab_str = '\n'.join(word for word, count in counter.most_common(500000))
+    with open('librispeech-vocab-500k.txt', 'w') as fout:
```

Comment:
I don't see that file under `data/lm`, did you forgot to `git add` ?

Reply:
No, it'll be generated alongside `lm.binary` by `generate_lm.py`.